components/molecules/charts/schedule/helper/columns.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * Column
+ */
+
+// React
+import React from 'react'
```

Comment:
@Ojaghi1991 Space below

// Style
...

No Proptypes on this component - there should be?

Reply:
Columns is not Component, it's helper function to generate column